cypress/integration/ete-okta/reset_password.cy.ts

```diff
@@ -1,5 +1,13 @@
+import { Status } from '../../../src/server/models/okta/User';
 import { randomPassword } from '../../support/commands/testUser';
 
+const breachCheck = () => {
```

Comment:
might be nice to make this a command, then we can benefit from this in all the other tests we have this `intercept` 🙂

Reply:
Good idea, but this should probably go into another PR as I don't want to pollute this PR with changes to that.